src/index.js

```diff
@@ -40,7 +40,7 @@ export default function copy(options = {}) {
 
   return {
     name: 'copy',
-    [hook]: async () => {
+    async [hook]() {
```

Comment:
This was necessary because the `this` keyword was not available with the unbound arrow function syntax.

See this issue for more details: https://github.com/rollup/rollup/issues/1518#issuecomment-374727013